### ISSUE TITLE
Feature request: class attribute on fallback image in picture tag

### ISSUE THREAD

**Issue by AlexBa** [alexjoverm/v-lazy-image#47]
Hello,
I would like to use custom CSS classes on the fallback image in the picture tag. As far I see it in the code, it's currently not possible to do this. I imagine something like:

```
<v-lazy-image :src="require('~/assets/images/logos/aws.png')" img-class="img-fluid d-block" use-picture>
    <source :srcset="require('~/assets/images/logos/aws.png?webp')" type="image/webp" />
    <source :srcset="require('~/assets/images/logos/aws.png')" type="image/png" />
</v-lazy-image>
```

Which will be rendered as:
```
<picture>
    <source srcset="/_nuxt/assets/images/logos/aws.webp" type="image/webp"> 
    <source srcset="/_nuxt/assets/images/logos/aws.png" type="image/png">
    <img src="/_nuxt/assets/images/logos/aws.png" class="img-fluid d-block v-lazy-image v-lazy-image-loaded">
</picture>
```

Would this be possible? Does the solution make sense or will be there any problems in the future?

Thank you in advance!

**Comment by highoncarbs** [comment]
Hey ! Got anything on this ?

**Comment by alexjoverm** [comment]
Yeah... That's just how Vue 2 works, but maybe can be done via a prop. If you want you can go for a PR :)

**Comment by alexjoverm** [comment]
Actually, this is fixed and should work in latest version > 2.1.0
